docsrc/allblocks.nim

```diff
@@ -66,6 +65,25 @@ nimibCode:
   nbCodeSkip:
     exit() # even this won't execute!
 
+nbCodeBlock: "nbCodeInBlock"
+nbText: """
+Sometimes, you want to show similar code snippets with the same variable names and you need to declare twice the same variables.
+You can do so with `nbCodeInBlock` which nests your snippets inside invisible blocks.
```

Comment:
```suggestion
You can do so with `nbCodeInBlock` which nests your snippet inside a block.
```
Good explanation, and I like the comments in the code below :D. I would just rephrase this sentence to not use "invisible".